app/models/bot/alegre.rb

```diff
@@ -36,6 +36,14 @@ def delete_analysis_from_similarity_index
     end
   end  
 
+  def self.default_model
+    'xlm-r-bert-base-nli-stsb-mean-tokens'
+  end
+
+  def self.indian_model
+    'indian-sbert'
+  end
+
```

Comment:
What about ElasticSearch? Are we dropping it completely?

Reply:
Also, it could be just a constant instead of a method, right?

Reply:
We are dropping elasticsearch as a default - it can obviously be passed in - I'll change this to a constant.